### ISSUE TITLE
Cannot parse babel preset's url

### ISSUE THREAD

**Issue by kitak** [bitjourney/ci-npm-update#50]
When babel preset package is included in the dependencies, the `Cannot parse git repository URL` error will occur.

```
Unexpected errors caught: Error: Cannot parse git repository URL: https://github.com/babel/babel/tree/master/packages/babel-preset-es2015
    at Error (native)
    at Function.parseUrl (/home/kitak/local/node-v6.6.0/lib/node_modules/ci-npm-update/lib/github.js:31:19)
    at Function.fixupUrl (/home/kitak/local/node-v6.6.0/lib/node_modules/ci-npm-update/lib/compare_view.js:14:39)
    at new GitHubCompareView (/home/kitak/local/node-v6.6.0/lib/node_modules/ci-npm-update/lib/compare_view.js:8:48)
    at result.push.npm_config_1.NpmConfig.getFromRegistry.then (/home/kitak/local/node-v6.6.0/lib/node_modules/ci-npm-update/lib/shrink_wrap.js:70:24)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by gfx** [comment]
That's because it is not a git repository URL.

Okay, ci-npm-update should update invalid URLs, but is is a bug in babel-preset-es2015's package.json.

**Comment by gfx** [comment]
Released 1.0.8 with the fix of this issue. Thanks!
